script/migrations/18-fund2.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import "../../src/wallet/KintoWalletFactory.sol";
+import "../../src/wallet/KintoWallet.sol";
+import "../../src/paymasters/SponsorPaymaster.sol";
+import "../../src/KintoID.sol";
+
+import "../../test/helpers/Create2Helper.sol";
+import "../../test/helpers/ArtifactsReader.sol";
+import "../../test/helpers/UUPSProxy.sol";
+
+import "forge-std/Script.sol";
+import "forge-std/console.sol";
+
+contract KintoMigration18DeployScript is Create2Helper, ArtifactsReader {
+    using ECDSAUpgradeable for bytes32;
+
+    KintoWalletFactoryV5 _factoryImpl;
```

Comment:
i feel like at this point, the `KintoWalletFactory` maaaaay be renamed to something else since the factory is not just to create wallets, right?
- `KintoPeriphery`
- `KintoHelper`
- `KintoFactory`
thoughts?

Reply:
lets do this in a follow up PR